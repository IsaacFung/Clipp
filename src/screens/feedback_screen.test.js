import React from "react";
import renderer from 'react-test-renderer';

import FeedbackScreen from './feedback_screen'

it('renders correctly', () => {
    const tree = renderer.create(<FeedbackScreen/>).toJSON();
    expect(tree).toMatchSnapshot();
});