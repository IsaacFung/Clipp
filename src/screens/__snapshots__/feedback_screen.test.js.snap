// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
        "height": 1334,
      },
      Object {
        "alignItems": "center",
        "backgroundColor": "#ffffff",
        "flex": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <RCTScrollView
    automaticallyAdjustContentInsets={false}
    contentContainerStyle={
      Array [
        Object {
          "padding": 10,
        },
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "flex": 1,
          "justifyContent": "center",
        },
      ]
    }
    contentInset={
      Object {
        "bottom": 0,
      }
    }
    enableAutomaticScroll={true}
    enableOnAndroid={false}
    enableResetScrollToCoords={true}
    extraHeight={75}
    extraScrollHeight={0}
    getScrollResponder={[Function]}
    handleOnScroll={[Function]}
    keyboardDismissMode="interactive"
    keyboardOpeningTime={250}
    keyboardShouldPersistTaps="handled"
    keyboardSpace={0}
    onScroll={[Function]}
    padder={true}
    resetKeyboardSpace={[Function]}
    resetScrollToCoords={
      Object {
        "x": 0,
        "y": 0,
      }
    }
    scrollEventThrottle={1}
    scrollForExtraHeightOnAndroid={[Function]}
    scrollIntoView={[Function]}
    scrollToEnd={[Function]}
    scrollToFocusedInput={[Function]}
    scrollToPosition={[Function]}
    showsVerticalScrollIndicator={true}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      }
    }
    update={[Function]}
    viewIsInsideTabBar={false}
  >
    <View>
      <Image
        source={1}
        style={
          Object {
            "height": 144,
            "margin": "auto",
            "resizeMode": "center",
          }
        }
      />
      <Text
        style={
          Array [
            Object {
              "color": "#000",
              "fontFamily": "System",
              "fontSize": 16,
            },
            Object {
              "paddingLeft": 16,
              "paddingRight": 16,
            },
          ]
        }
        uppercase={false}
      >
        Please leave some feedback for the app. We appreciate any comments or suggestions to help us improve your experience!
      </Text>
      <TextInput
        allowFontScaling={true}
        bordered={true}
        editable={true}
        multiline={true}
        onChangeText={[Function]}
        placeholder="Please leave some feedback..."
        placeholderTextColor="#575757"
        rejectResponderTermination={true}
        rowSpan={5}
        style={
          Object {
            "borderBottomWidth": 1,
            "borderColor": "#D9D5DC",
            "borderLeftWidth": 1,
            "borderRightWidth": 1,
            "borderTopWidth": 1,
            "color": "#000",
            "fontSize": 15,
            "height": 125,
            "marginBottom": 16,
            "marginRight": 8,
            "marginTop": 16,
            "paddingLeft": 10,
            "paddingRight": 5,
            "textAlignVertical": "top",
            "width": "90%",
          }
        }
        textContentType="none"
        underlineColorAndroid="rgba(0,0,0,0)"
        value=""
      />
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#5cb85c",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 5,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "borderWidth": undefined,
            "elevation": 2,
            "flexDirection": "row",
            "height": 60,
            "justifyContent": "space-between",
            "marginBottom": 16,
            "marginTop": 16,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingTop": 6,
            "shadowColor": undefined,
            "shadowOffset": undefined,
            "shadowOpacity": undefined,
            "shadowRadius": undefined,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
                "color": "#fff",
                "fontFamily": "System",
                "fontSize": 22,
                "marginLeft": 0,
                "marginRight": 0,
                "paddingLeft": 16,
                "paddingRight": 16,
              },
              Object {
                "color": "#FFFFFF",
                "fontSize": 20,
                "fontWeight": "bold",
                "textTransform": "uppercase",
              },
            ]
          }
          uppercase={false}
        >
          Submit
        </Text>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
